backend/schema/Clips.js

```diff
@@ -85,8 +85,10 @@ cube(`Clips`, {
       type: `string`,
       case: {
         when: [
-            { sql: `${CUBE}.age = 'ungur_unglingur'`, label: `13-17` },
-            { sql: `${CUBE}.age = 'unglingur'`, label: `18-19` },
+            { sql: `${CUBE}.age < 10 and ${CUBE}.age > 0`, label: `0-9` },
+            { sql: `${CUBE}.age < 18 and ${CUBE}.age > 9`, label: `10-19` },
```

Comment:
Hey! Can you change this to 0-12 and 13-19

Then it fits well with how we are defining the groups for ASR dev

Reply:
Ofc!